nodes/src/nodes/ocr/IInstance.py

```
@@ -44,12 +44,13 @@ def extract_tables_from_image(self, image_data: bytes, table_callback=None):
         :param image_data: Image in bytes
         :param table_callback: Function to call with each extracted Markdown table
         """
+
         # Write diagnostics to a file since print() goes to DAP
         def _diag(msg):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_diag`

Add return type annotation: `None`

(ANN202)